### DISCUSSION TITLE
Disable Creation of files test_log.INFO, test_log.DEBUG, test_summary.yaml

### DISCUSSION THREAD

**Discussion by Harshith-GRL** [google/mobly#915] [Q&A]
I am trying to find a way to disable the creation of three files namely test_log.INFO, test_log.DEBUG, test_summary.yaml done by mobly . I am trying to find resource in the documentation but no luck at the moment. DO i have to use some workaround for it that I am not aware of ?

**Comment by xpconanfan** [comment]
That's not possible.
Why are you trying to do this.

**Reply by Harshith-GRL** [reply]
In our requirement we are logging the data in custom way and having this is causing some duplicate data being present in system and these files along with our data is taking up alot of space.
can we an bring this option to the framework to disable and enable these file creations?

**Reply by xpconanfan** [reply]
You can easily remove these files as part of your lab management.

This is not something we plan to do
